<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>292</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Conversor de Monedas</string>
  </property>
  <property name="windowIcon">
   <iconset theme="view-refresh"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="monedaEntrante">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dólar estadounidense (USD)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro (EUR)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yen japonés (JPY)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Libra esterlina (GBP)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Franco suizo (CHF)</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="monedaSaliente">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dólar estadounidense (USD)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro (EUR)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yen japonés (JPY)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Libra esterlina (GBP)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Franco suizo (CHF)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="cerrarVentana">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="limpiar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="simboloEntrante">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>$</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_aniadir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir Moneda</string>
    </property>
   </widget>
   <widget class="QLabel" name="hora">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
